### ISSUE TITLE
Files tree refresh in crossover

### ISSUE THREAD

**Issue by zaynyatyi** [fdorg/flashdevelop#958]
On Mac OS when creating new file, folder or changing filenames from external tool files tree in FD is not refreshing. I have to refresh it manually.

**Comment by elsassph** [comment]
You need to install FlashDevelop Bridge on the Mac/Linux side to help FD know about filechanges.
http://www.flashdevelop.org/wikidocs/index.php?title=Bridge

**Comment by zaynyatyi** [comment]
It's installed and shared drive path configured. Should i configure it to folder where my projects are placed?

**Comment by elsassph** [comment]
Is the project located in the shared location?

**Comment by zaynyatyi** [comment]
Yes. Is there some way i can debug bridge? On wiki i see "You should now see messages from Bridge in the OutputPanel.", but there is no messages from my bridge.

**Comment by Meychi** [comment]
Sometimes the connections are blocked or something for sometime after install but if you open FD via opening FD project file from Finder, the connection should start to work. I don't know reason for this.

**Comment by zaynyatyi** [comment]
Tried starting FD from finder using project file, but without any effect

**Comment by elsassph** [comment]
@meychi, should there still be a trace in the Output panel indicating which party is being monitored by the bridge?

**Comment by Meychi** [comment]
Just to make sure, did you enable the bridge from the settings? I have accidentally missed that few times.

FlashDevelop currently indicates if the connection succeeded or failed if the bridge plugin is enabled.

**Comment by Meychi** [comment]
@zaynyatyi Is this working for you?
